spring-integration-hazelcast/src/main/java/org/springframework/integration/hazelcast/leader/LeaderInitiator.java

```diff
@@ -263,30 +299,36 @@ protected class LeaderSelector implements Callable<Void> {
 
 		protected final String role = LeaderInitiator.this.candidate.getRole();
 
-		private volatile boolean locked = false;
+		private volatile boolean leader = false;
 
 		@Override
 		public Void call() {
 			try {
 				while (isRunning()) {
 					try {
-						if (LeaderInitiator.this.lock.isLocked()) {
+						if (logger.isDebugEnabled()) {
+							logger.debug("Am I the leader (" + LeaderInitiator.this.candidate.getRole() + ") ? "
+									+ this.leader);
```

Comment:
I don't see a reason in this debug message.

Reply:
The reason is to see the thread heartbeat activity when debugging. If it is not needed I can remove it.

Reply:
OK. No problem! Then `trace()` might be better 😄